contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mir/jobs.pxu

```diff
@@ -18,11 +18,13 @@ plugin: shell
 flags: also-after-suspend
 user: root
 depends: mir/check-ubuntu-frame-launching-auto
-imports: 
+imports:
     from com.canonical.certification import snap
     from com.canonical.plainbox import manifest
+    from com.canonical.certification import connections
 requires:
     snap.name == "graphics-test-tools"
+    connections.plug in ["graphics-test-tools:graphics-core20", "graphics-test-tools:graphics-core22", "graphics-test-tools:gpu-2404"]
```

Comment:
I suggest we remove this `require condition`, so the test would failed if user did not connect to correct slot, instead of skip this test.

Reply:
Pushed the fix. Thanks.